CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 ## Unreleased
 
 - `documentation` fixed the `utils/window` example
+- `map` added possibility for users to specify zoomlevel when zooming to their current position on the `aui-leaflet-locate-control`.
```

Comment:
It's not the users, but the developer that can specify the zoomlevel. ;-)
Perhaps something more like 'added possibility to set zoom level of the `aui-leaflet-locate-control`?